doc/source/getting_started/index.rst

```diff
@@ -45,30 +45,19 @@ To launch Fluent from PyFluent, use the ``launch_fluent`` method:
 .. code:: python
 
   import ansys.fluent.core as pyfluent
-  solver_session = pyfluent.launch_fluent(precision="double", processor_count=2, mode="solver")
-  solver_session.check_health()
-
-On Windows systems the environment variable ``AWP_ROOT<ver>``, is configured
```

Comment:
Just noticed you deleted these two paragraphs.  Did you relocate the information or intentionally delete it because it's no longer correct?

Reply:
just scroll down slightly

Reply:
Is it in a different file or something?  I'm not seeing it.  Blind I guess.

Reply:
@dnwillia-work sorry I was looking at the two lines of Python. I see now. Hold on, ...

Reply:
This must have been a merge issue. These lines are a recent addition.

Reply:
Reinstated.